tests/sync.cpp

```diff
@@ -64,8 +64,8 @@ TEST_CASE("sync: log-in", "[sync]") {
     SECTION("Can log in") {
         std::atomic<int> error_count(0);
         auto session = sync_session(server, "user", "/test",
-                                    [&](auto...) { return s_test_token; },
-                                    [&](auto...) { ++error_count; });
+                                    [&](const std::string&, const std::string&) { return s_test_token; },
```

Comment:
This is unfortunate. Which compiler version doesn't support variadic generic lambdas?